lib/entities/function.ts

```diff
@@ -14,5 +14,6 @@ export type FunctionProps = {
   name: string
   description: string
   path: string
+  accepts: string[]
```

Comment:
should this be optional as well?

```suggestion
  accepts?: string[]
```